packages/ui/src/common/components/BlockTime/BlockInfo.tsx

```diff
@@ -8,7 +8,6 @@ import { Colors } from '@/common/constants'
 import { useNetworkEndpoints } from '@/common/hooks/useNetworkEndpoints'
 import { formatTokenValue } from '@/common/model/formatters'
 import { Block } from '@/common/types'
-
```

Comment:
Please remove the unnecessary changes to this file to prevent potential conflicts later on.